homeassistant/components/fritz/strings.json

```diff
@@ -1,4 +1,13 @@
 {
+  "common": {
+    "data_description": {
+      "host": "The hostname or IP address of your FRITZ!Box router.",
+      "port": "Leave it empty to use the default port.",
+      "username": "Username for the FRITZ!Box.",
+      "password": "Password for the FRITZ!Box.",
+      "ssl": "Use SSL to connect to the FRITZ!Box."
```

Comment:
maybe we should align these descriptions with that one from the [docs](https://www.home-assistant.io/integrations/fritz#configuration)? what do you think?